ova/generate_ova.sh

```diff
@@ -56,8 +57,8 @@ function build_ova() {
 
   #Download filebeat.yml and enable geoip
   if [ ${ELK_MAJOR} -eq 7 ]; then
-      curl -so Config_files/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/v${WAZUH_VERSION}/extensions/filebeat/7.x/filebeat.yml
-      sed -i "s|#pipeline: geoip|pipeline: geoip|" Config_files/filebeat.yml
+      curl -so config_files/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/v${WAZUH_VERSION}/extensions/filebeat/7.x/filebeat.yml
```

Comment:
Be careful with this line. You are downloading the `filebeat.yml` file in `config_files` and this directory doesn't exist. Did you mean `Config_files` ?

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/2629f18c2d161e7cfe70b71d39522548b5459f55